src/runtime_strings_extra.js

```diff
@@ -77,6 +77,37 @@ function UTF16ToString(ptr, maxBytesToRead) {
 #endif // TEXTDECODER
 }
 
+function UTF16ToStringWithLength(ptr, lengthInBytes) {
```

Comment:
Please move all the added code from this file into file `src/library_strings.js`. This file exists only for historical compatibility reasons, but all new code can be added to `src/library_strings.js`.

Reply:
accepted